app/models/comment.rb

```diff
@@ -0,0 +1,9 @@
+class Comment < ApplicationRecord
+  belongs_to :post
+  belongs_to :user
+
+  def self.update_comments_counter(post)
+    comments_count = Comment.where(post:).count
+    post.update(CommentsCounter: comments_count)
```

Comment:
- [ ] As stated before, use this line `post.increment!(:comments_counter)` to update the comment counter